### DISCUSSION TITLE
Shape Functions and Node

### DISCUSSION THREAD

**Discussion by maxnezdyur** [idaholab/moose#23841] [Q&A General] [answered]
Within an element, are the order of the nodes and the order of the shape functions the same? If I know that a node is the third node of the element, does the third "phi" correspond to that node?

**Comment by jiangwen84** [comment, marked as answer]
> Within an element, are the order of the nodes and the order of the shape functions the same? If I know that a node is the third node of the element, does the third "phi" correspond to that node?

That should be the case. 

@maxnezdyur btw, are you a hurricane fan?

**Reply by maxnezdyur** [reply]
Perfect, that's what I was hoping for. Of course, always supporting my Canes.
